File: M. Rivqi Al Varras_22004_source code unit IV/ratarata.py
numbers = [None for i in range(100)]
count = 10
sum = 0
average = 0.0
print("\nMasukan 100 bilangan : ")
print()
for i in range(0, 100):
    print("Data ke - ",i+1, ">");numbers[i]=int(input())
    sum+=numbers[i]
    average=float(sum)/count
print("\n Nilai rata-ratanya adalah: ",average,"\n")

File: M. Rivqi Al Varras_22004_source code unit IV/ratarata.c
#include <stdio.h>
#include <stdlib.h>
 
int main(){
   int i;
   int numbers[100];
   int count = 100;
   long sum = 0L;
   float average = 0.0f;
   printf("Masukan 100 bilangan: \n");
   for (i = 0; i < count; i ++){
       printf("Data ke - %2d> ",i+1);
       scanf("%d", &numbers[i]);
       sum += numbers[i];
   }
   average = (float)sum/count;
   printf("\n Nilai rata-ratanya adalah: %f\n", average);
  
   system("pause");
   return 0;
}